From 5a8d54ee7cc283ec6f3902e35d00437d8b176d3a Mon Sep 17 00:00:00 2001
From: David Gumberg <davidzgumberg@gmail.com>
Date: Sat, 1 Nov 2014 22:15:01 -0700
Subject: [PATCH] Update appdata to comply with new specification

http://blogs.gnome.org/hughsie/2014/05/02/appdata-meet-spdx-spdx-meet-appdata/

Please let me know if I messed up on the project licensing, I was unsure of the vdrift-data license, as the only thing I could find was that "vdrift" was gplv3. The fedora repo shows that "vdrift-data" is GPLv3+ so I went with that.
---
 vdrift.appdata.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/vdrift.appdata.xml b/vdrift.appdata.xml
index 9fd1cca7f..5bcb28be9 100644
--- a/vdrift.appdata.xml
+++ b/vdrift.appdata.xml
@@ -2,7 +2,8 @@
 <!-- Copyright 2014 Sergej Forat <core13@gmx.net> -->
 <application>
  <id type="desktop">vdrift.desktop</id>
- <licence>CC0</licence>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0+</project_license>
  <name>VDrift</name>
  <summary>Car Racing Simulator</summary>
  <description>
